recipes/pycudadecon/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pycudadecon" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tlambert03/pycudadecon.git
+  git_rev: v{{ version }}
+
+build:
+  skip: True  # [osx]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
```

```suggestion
    - python >=3.7
```

ok, now we're back to [getting python version errors](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=330449&view=logs&j=232b7f5e-8d64-5438-fada-8478f379d226&t=e8516d7e-1153-5090-5807-53ae833fddd8&l=1593): `Package 'pycudadecon' requires a different Python: 3.6.13 not in '>=3.7'` ... so lets try this again.